extension/inet/include/html_charref.hpp

```diff
@@ -0,0 +1,4552 @@
+		mapped_strings[string_t(owned_strings[129])] = 330;
+		mapped_strings[string_t(owned_strings[130])] = 208;
+		mapped_strings[string_t(owned_strings[131])] = 20;
+		mapped_strings[string_t(owned_strings[132])] = 201;
+		mapped_strings[string_t(owned_strings[133])] = 201;
+		mapped_strings[string_t(owned_strings[134])] = 28;
+		mapped_strings[string_t(owned_strings[135])] = 202;
+		mapped_strings[string_t(owned_strings[136])] = 202;
+		mapped_strings[string_t(owned_strings[137])] = 106;
+		mapped_strings[string_t(owned_strings[138])] = 278;
+		mapped_strings[string_t(owned_strings[139])] = 120072;
+		mapped_strings[string_t(owned_strings[140])] = 20;
+		mapped_strings[string_t(owned_strings[141])] = 200;
+		mapped_strings[string_t(owned_strings[142])] = 8712;
+		mapped_strings[string_t(owned_strings[143])] = 27;
+		mapped_strings[string_t(owned_strings[144])] = 9723;
+		mapped_strings[string_t(owned_strings[145])] = 9643;
+		mapped_strings[string_t(owned_strings[146])] = 28;
+		mapped_strings[string_t(owned_strings[147])] = 120124;
+		mapped_strings[string_t(owned_strings[148])] = 917;
+		mapped_strings[string_t(owned_strings[149])] = 1086;
+		mapped_strings[string_t(owned_strings[150])] = 8770;
+		mapped_strings[string_t(owned_strings[151])] = 8652;
+		mapped_strings[string_t(owned_strings[152])] = 849;
+		mapped_strings[string_t(owned_strings[153])] = 10867;
+		mapped_strings[string_t(owned_strings[154])] = 919;
+		mapped_strings[string_t(owned_strings[155])] = 20;
+		mapped_strings[string_t(owned_strings[156])] = 203;
+		mapped_strings[string_t(owned_strings[157])] = 8707;
+		mapped_strings[string_t(owned_strings[158])] = 851;
+		mapped_strings[string_t(owned_strings[159])] = 1060;
+		mapped_strings[string_t(owned_strings[160])] = 120073;
+		mapped_strings[string_t(owned_strings[161])] = 972;
+		mapped_strings[string_t(owned_strings[162])] = 9642;
+		mapped_strings[string_t(owned_strings[163])] = 120125;
+		mapped_strings[string_t(owned_strings[164])] = 870;
+		mapped_strings[string_t(owned_strings[165])] = 8497;
+		mapped_strings[string_t(owned_strings[166])] = 8497;
+		mapped_strings[string_t(owned_strings[167])] = 102;
+		mapped_strings[string_t(owned_strings[168])] = 62;
+		mapped_strings[string_t(owned_strings[169])] = 62;
+		mapped_strings[string_t(owned_strings[170])] = 91;
+		mapped_strings[string_t(owned_strings[171])] = 988;
+		mapped_strings[string_t(owned_strings[172])] = 286;
+		mapped_strings[string_t(owned_strings[173])] = 29;
+		mapped_strings[string_t(owned_strings[174])] = 284;
+		mapped_strings[string_t(owned_strings[175])] = 1043;
+		mapped_strings[string_t(owned_strings[176])] = 28;
+		mapped_strings[string_t(owned_strings[177])] = 120074;
+		mapped_strings[string_t(owned_strings[178])] = 8921;
+		mapped_strings[string_t(owned_strings[179])] = 12012;
+		mapped_strings[string_t(owned_strings[180])] = 8805;
+		mapped_strings[string_t(owned_strings[181])] = 8923;
+		mapped_strings[string_t(owned_strings[182])] = 880;
```

Comment:
Can we please regenerate the header file, this entry is wrong:
`"GreaterFullEqual;",                // 182` + `mapped_strings[string_t(owned_strings[182])] = 880;`
Does not match:
`"&GreaterFullEqual;": { "codepoints": [8807], "characters": "\u2267" }`

Lists like these have to be generated, not only does creating this by hand take a long time, it's also only natural that human errors like these creep in

Reply:
This list was generated by parsing this json [file](https://html.spec.whatwg.org/entities.json) with the database. Over time, I've made a few changes to the file, so it's possible that I accidentally modified some values. Thank you for taking the time to review them. I'll regenerate the list and take a closer look to identify any differences.

Reply:
I apologize, I jumped to conclusions

Reply:
I wrote a quick script, I think a multi-cursor edit accidentally caused the last digit of some to be removed:
```
entity &Aring; codepoint (19) is wrong, should have been 197
entity &Atilde codepoint (19) is wrong, should have been 195
entity &Auml; codepoint (19) is wrong, should have been 196
entity &Barwed; codepoint (896) is wrong, should have been 8966
entity &Bernoullis; codepoint (849) is wrong, should have been 8492
entity &Bopf; codepoint (12012) is wrong, should have been 120121
entity &Bumpeq; codepoint (878) is wrong, should have been 8782
entity &COPY; codepoint (16) is wrong, should have been 169
entity &CapitalDifferentialD; codepoint (851) is wrong, should have been 8517
entity &Ccedil codepoint (19) is wrong, should have been 199
entity &Cconint; codepoint (875) is wrong, should have been 8752
entity &CenterDot; codepoint (18) is wrong, should have been 183
entity &CircleDot; codepoint (885) is wrong, should have been 8857
entity &CircleTimes; codepoint (885) is wrong, should have been 8855
entity &CloseCurlyQuote; codepoint (821) is wrong, should have been 8217
entity &Congruent; codepoint (880) is wrong, should have been 8801
entity &Copf; codepoint (845) is wrong, should have been 8450
entity &Cross; codepoint (1079) is wrong, should have been 10799
entity &CupCap; codepoint (878) is wrong, should have been 8781
entity &DJcy; codepoint (102) is wrong, should have been 1026
entity &Dagger; codepoint (822) is wrong, should have been 8225
entity &Dcaron; codepoint (27) is wrong, should have been 270
entity &Delta; codepoint (91) is wrong, should have been 916
entity &DiacriticalDot; codepoint (72) is wrong, should have been 729
entity &DiacriticalTilde; codepoint (73) is wrong, should have been 732
entity &Dopf; codepoint (12012) is wrong, should have been 120123
entity &DotEqual; codepoint (878) is wrong, should have been 8784
entity &DoubleDownArrow; codepoint (865) is wrong, should have been 8659
entity &DoubleLeftTee; codepoint (1098) is wrong, should have been 10980
entity &DoubleLongRightArrow; codepoint (1023) is wrong, should have been 10233
entity &DoubleUpArrow; codepoint (865) is wrong, should have been 8657
entity &DownArrow; codepoint (859) is wrong, should have been 8595
entity &DownBreve; codepoint (78) is wrong, should have been 785
entity &DownLeftVector; codepoint (863) is wrong, should have been 8637
entity &DownRightVector; codepoint (864) is wrong, should have been 8641
entity &DownTeeArrow; codepoint (861) is wrong, should have been 8615
entity &Dstrok; codepoint (27) is wrong, should have been 272
entity &ETH; codepoint (20) is wrong, should have been 208
entity &Ecaron; codepoint (28) is wrong, should have been 282
entity &Ecy; codepoint (106) is wrong, should have been 1069
entity &Egrave codepoint (20) is wrong, should have been 200
entity &Emacr; codepoint (27) is wrong, should have been 274
entity &Eogon; codepoint (28) is wrong, should have been 280
entity &Equal; codepoint (1086) is wrong, should have been 10869
entity &Escr; codepoint (849) is wrong, should have been 8496
entity &Euml codepoint (20) is wrong, should have been 203
entity &ExponentialE; codepoint (851) is wrong, should have been 8519
entity &FilledSmallSquare; codepoint (972) is wrong, should have been 9724
entity &ForAll; codepoint (870) is wrong, should have been 8704
entity &GJcy; codepoint (102) is wrong, should have been 1027
entity &Gamma; codepoint (91) is wrong, should have been 915
entity &Gcedil; codepoint (29) is wrong, should have been 290
entity &Gdot; codepoint (28) is wrong, should have been 288
entity &Gopf; codepoint (12012) is wrong, should have been 120126
entity &GreaterFullEqual; codepoint (880) is wrong, should have been 8807
entity &GreaterSlantEqual; codepoint (1087) is wrong, should have been 10878
entity &Hat; codepoint (9) is wrong, should have been 94
entity &HilbertSpace; codepoint (845) is wrong, should have been 8459
entity &Hscr; codepoint (845) is wrong, should have been 8459
entity &HumpEqual; codepoint (878) is wrong, should have been 8783
entity &IOcy; codepoint (102) is wrong, should have been 1025
entity &Icirc codepoint (20) is wrong, should have been 206
entity &Idot; codepoint (30) is wrong, should have been 304
entity &Igrave; codepoint (20) is wrong, should have been 204
entity &ImaginaryI; codepoint (852) is wrong, should have been 8520
entity &Integral; codepoint (874) is wrong, should have been 8747
entity &InvisibleTimes; codepoint (829) is wrong, should have been 8290
entity &Iota; codepoint (92) is wrong, should have been 921
entity &Iukcy; codepoint (103) is wrong, should have been 1030
entity &Jcirc; codepoint (30) is wrong, should have been 308
entity &Jopf; codepoint (12012) is wrong, should have been 120129
entity &Jukcy; codepoint (102) is wrong, should have been 1028
entity &Kappa; codepoint (92) is wrong, should have been 922
entity &Kfr; codepoint (12007) is wrong, should have been 120078
entity &LJcy; codepoint (103) is wrong, should have been 1033
entity &Lacute; codepoint (31) is wrong, should have been 313
entity &Laplacetrf; codepoint (846) is wrong, should have been 8466
entity &Lcedil; codepoint (31) is wrong, should have been 315
entity &LeftArrow; codepoint (859) is wrong, should have been 8592
entity &LeftCeiling; codepoint (896) is wrong, should have been 8968
entity &LeftDownVector; codepoint (864) is wrong, should have been 8643
entity &LeftRightArrow; codepoint (859) is wrong, should have been 8596
entity &LeftTeeArrow; codepoint (861) is wrong, should have been 8612
entity &LeftTriangleBar; codepoint (1070) is wrong, should have been 10703
entity &LeftUpTeeVector; codepoint (1059) is wrong, should have been 10592
entity &LeftVector; codepoint (863) is wrong, should have been 8636
entity &Leftrightarrow; codepoint (866) is wrong, should have been 8660
entity &LessGreater; codepoint (882) is wrong, should have been 8822
entity &LessTilde; codepoint (881) is wrong, should have been 8818
entity &Lleftarrow; codepoint (866) is wrong, should have been 8666
entity &LongLeftRightArrow; codepoint (1023) is wrong, should have been 10231
entity &Longleftrightarrow; codepoint (1023) is wrong, should have been 10234
entity &LowerLeftArrow; codepoint (860) is wrong, should have been 8601
entity &Lsh; codepoint (862) is wrong, should have been 8624
entity &Map; codepoint (1050) is wrong, should have been 10501
entity &Mellintrf; codepoint (849) is wrong, should have been 8499
entity &Mopf; codepoint (12013) is wrong, should have been 120132
entity &NJcy; codepoint (103) is wrong, should have been 1034
entity &Ncedil; codepoint (32) is wrong, should have been 325
entity &NegativeThickSpace; codepoint (820) is wrong, should have been 8203
entity &NestedGreaterGreater; codepoint (881) is wrong, should have been 8811
entity &Nfr; codepoint (12008) is wrong, should have been 120081
entity &Nopf; codepoint (846) is wrong, should have been 8469
entity &NotCupCap; codepoint (881) is wrong, should have been 8813
entity &NotEqual; codepoint (880) is wrong, should have been 8800
entity &NotGreater; codepoint (881) is wrong, should have been 8815
entity &NotGreaterTilde; codepoint (882) is wrong, should have been 8821
entity &NotLeftTriangle; codepoint (893) is wrong, should have been 8938
entity &NotLess; codepoint (881) is wrong, should have been 8814
entity &NotRightTriangle; codepoint (893) is wrong, should have been 8939
entity &NotSquareSupersetEqual; codepoint (893) is wrong, should have been 8931
entity &NotSucceeds; codepoint (883) is wrong, should have been 8833
entity &NotTilde; codepoint (876) is wrong, should have been 8769
entity &NotTildeTilde; codepoint (877) is wrong, should have been 8777
entity &Ntilde codepoint (20) is wrong, should have been 209
entity &OElig; codepoint (33) is wrong, should have been 338
entity &Ocirc codepoint (21) is wrong, should have been 212
entity &Odblac; codepoint (33) is wrong, should have been 336
entity &Ograve; codepoint (21) is wrong, should have been 210
entity &Omicron; codepoint (92) is wrong, should have been 927
entity &OpenCurlyQuote; codepoint (821) is wrong, should have been 8216
entity &Oslash codepoint (21) is wrong, should have been 216
entity &Otilde; codepoint (21) is wrong, should have been 213
entity &Ouml; codepoint (21) is wrong, should have been 214
entity &OverBracket; codepoint (914) is wrong, should have been 9140
entity &Pcy; codepoint (105) is wrong, should have been 1055
entity &Pi; codepoint (92) is wrong, should have been 928
entity &Popf; codepoint (847) is wrong, should have been 8473
entity &PrecedesEqual; codepoint (1092) is wrong, should have been 10927
entity &Prime; codepoint (824) is wrong, should have been 8243
entity &Proportional; codepoint (873) is wrong, should have been 8733
entity &QUOT codepoint (3) is wrong, should have been 34
entity &Qopf; codepoint (847) is wrong, should have been 8474
entity &REG codepoint (17) is wrong, should have been 174
entity &Rang; codepoint (1021) is wrong, should have been 10219
```

Affects a total of 135 entries

Reply:
Happens to the best of us, I think Mark's idea for the csv is a great idea to help with avoiding this 👍